vignettes/computations.Rmd

```diff
@@ -0,0 +1,80 @@
+- **Come back rate** looks at the degree seeking students enrolled in each semester and tracks if the student re-enrolls (or comes back) by a given semester and removes exclusions.
+- **Return rate** looks at the degree seeking students enrolled in each semester and tracks if the student re-enrolls (or returns) or completes a credential by a given semester and removes exclusions. 
+
+
+|                  | Group of Students                                    | Retention Includes                                           | Retention Excludes |
+| ---------------- | ---------------------------------------------------- | ------------------------------------------------------------ | ------------------ |
+| Cohort Retention | Cohort, First-time Freshman and/or Transfer Students | Returned the specified semester                              | Exclusions         |
+| Come Back Rate   | All degree seeking students                          | Returned the specified semester                              | Exclusions         |
+| Return Rate      | All degree seeking students                          | Returned or completed a credential by the specified semester | Exclusions         |
+
+Retention rates also look at different time periods:
+
+- Fall to Spring - Student was enrolled at Fall Census and returned the following Spring term (second semester). For example enrolled in Fall 2020, returned Spring 2021.
+- Fall to Fall - Student was enrolled at Fall Census and returned the following fall term (fall 2). For example enrolled in Fall 2020, returned Fall 2021.
+- Returned 3rd Fall - Student was enrolled at Fall Census and returned in the fall 2 years later (fall 3). For example enrolled in Fall 2020, returned Fall 2022. This is used only for cohort retention.
+- Returned 4th Fall - Student was enrolled at Fall Census and returned in the fall 3 years later (fall 4). For example enrolled in Fall 2020, returned Fall 2023. This is used only for cohort retention.
+
+#### Year
+Data is reported by season (Fall, Spring, Summer) and Year. Season and Year together is referred to as Term. The year in the table and graph for retention metrics is the cohort year tied to the calendar year of the cohort term code (i.e. cohort term code - 202440, Fall 2024, is year 2024) or is the term year of the term being measured. The year is not the reporting year. Each retention metric has the 5 most recent years of data if it is available. If a retention metric needs 3 years to have complete data (returned 4th fall), then the metric will show the most recent complete data (Cohort retention returned 4th fall for the 2023 cohort will not be provided until Fall 2026). 
+
+### 150% Graduation Rate
+
+150% Graduation Rate tracks a cohort of students, after removing exclusions, for 6 years to see if the student earns a credential within 150% of the degree time (150% for a bachelor degree is 6 years) in a manner similar to IPEDS Graduation Rate. 
```

Comment:
We should add a discussion of how this impacts non bachelors students. We can catch that in the next round of reviews, but I think we'll need something.

Reply:
I have noted this an made a new issue. #36